tests/dns/dns-rcode/README.md

```diff
@@ -0,0 +1,4 @@
+Test the `dns.rcode` sticky buffer.
```

Comment:
nit: this is not a sticky buffer.  Probably just `Test the dns.rcode header value`?